app/assets/javascripts/application.js

```diff
@@ -8,4 +8,5 @@ require(['jquery', 'componentLoader', 'eventsWithPromises'], function ($, compon
 require(['jquery'], function ($) {
   require(['AdviserAjaxCall']);
   require(['RemoteAndFaceToFaceOptions']);
+  require(['WizardForm']);
```

Comment:
Generally on MAS projects we load JS components via our Dough Component Loader mechanism rather than through the manifest in this way. That would be our preferred way of doing this but since in this project there appears to already be a mixture of both methods it's probably not critical.